solver/interactiveSolver.py

```diff
@@ -1293,6 +1318,44 @@ def importDump(self, shmName):
                     else:
                         if loc in self.visitedLocations:
                             self.removeItemAt(self.locNameInternal2Web(loc.Name))
+
+            # Inventory
+            elif dataType == dataEnum["inventory"]:
+                # Clear collected items if loading from game state.
+                self.collectedItems.clear()
```

Comment:
same as above comment, doing that we break the regular tracker if a user stops using the auto tracker then want to continue manual tracking.
I'll add to reset all locations to unvisited when the auto tracker ends.